apps/publish/content/common.py

```diff
@@ -928,31 +928,36 @@ def _inherit_publish_schedule(self, original, updates, associated_item):
     def _update_picture_metadata(self, updates, original, updated):
         renditions = updated.get("renditions") or {}
         mapping = app.config.get("PICTURE_METADATA_MAPPING")
+        # Skip if no metadata mapping or no renditions
         if not mapping or not renditions:
             return
         try:
-            media_id = renditions["original"]["media"]
-        except (KeyError, TypeError):
-            return
-        if not media_id:
-            return
+            for rendition_key, rendition_data in renditions.items():
+                media_id = rendition_data.get("media")
```

Comment:
`rendition_data` could be null if some rendition can't be generated due to size, would be good to check that